### ISSUE TITLE
[v.3.0.0-beta.7][allure-jest][allure-js-commons] meta information is not passed to tests from the beforeEach fixture 

### ISSUE THREAD

**Issue by lifanov-rza** [allure-framework/allure-js#1075]
**Describe the bug**
All tests for related functionality are divided into directories, so in order not to duplicate code inside tests, we put all meta information (feature, link to documentation, etc.) in the beforeEach block. In the version on the master branch, this feature is present, however, in the new version, this functionality does not work, but we do not receive errors during code execution.

**To Reproduce**
1. create several tests using **JEST** in 1 file
2. add beforeEach block with adding meta information steps
3. run test
4. run allure serve
5. check meta info in allure

**Expected behavior**
Every tests have meta info

**Comment by baev** [comment]
@lifanov-rza please specify the test framework you're using.

**Comment by lifanov-rza** [comment]
> @lifanov-rza please specify the test framework you're using.

done

**Comment by lifanov-rza** [comment]
@baev Could you please provide me the information when are you going to fix this bug? It is very important for me, since the MP has already been prepared for moving to this reporter and only the presence of this problem blocks the release

**Comment by mkozakevich** [comment]
Same thing

I've tried some beta versions and found that problem is in the v3.0.0-beta.6 and later

Everything works fine in v3.0.0-beta.5

**How to reproduce:**

Add this code to any test

```
import {tag} from 'allure-js-commons';

beforeEach(async () => {
    await tag('some-tag');
});
```

Report will consist tag if you use v3.0.0-beta.5

```
{
    ...
    "labels": [
        {"name": "tag", "value": "some-tag"}
    ]
}
```

But report will consist only default labels if you use  v3.0.0-beta.6

**Comment by baev** [comment]
This issue is planned to be fixed for the final 3.0.0 release. 

WIP PR https://github.com/allure-framework/allure-js/pull/1041
